medicinal_plants/lib/src/navbar/category-navbar.dart

```diff
@@ -0,0 +1,127 @@
+// ignore_for_file: use_key_in_widget_constructors, library_private_types_in_public_api, avoid_unnecessary_containers, sized_box_for_whitespace
+
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:medicinal_plants/src/constants/colors.dart';
+
+class CategoryNavBar extends StatefulWidget {
+  @override
+  _CategoryNavBarState createState() => _CategoryNavBarState();
+}
+
+class _CategoryNavBarState extends State<CategoryNavBar> with SingleTickerProviderStateMixin {
+  late TabController _tabController;
+
+  final List<String> categories = ['Recommended', 'Cough', 'Fever', 'Headache'];
```

Comment:
we can move this into the "data" or perhaps a common,
but for now sa "data" lang sa